nav2_behaviors/src/behavior_server.cpp

```diff
@@ -142,7 +142,7 @@ void BehaviorServer::setupResourcesForBehaviorPlugins()
   std::string local_costmap_topic, global_costmap_topic;
   std::string local_footprint_topic, global_footprint_topic;
   std::string robot_base_frame;
-  double transform_tolerance;
+  double transform_tolerance {};
```

Comment:
Set to zero

Reply:
Done in 20acdca5436f2dcb918f9cceeb35730410b3b26b